sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -20,198 +20,198 @@ Sets properties on the SharePoint Online organization.
 
 ```powershell
 Set-SPOTenant
- [-ApplyAppEnforcedRestrictionsToAdHocRecipients <Boolean>]
+  [-AIBuilderModelScope <SyntexFeatureScopeValue>]
+ [-AIBuilderModelSelectedSitesIncludesContentCenters <Boolean>]
+ [-AIBuilderModelSelectedSitesList [String[]]]
+ [-AIBuilderModelSelectedSitesListOperation <SelectedSitesListOperations>]
+ [-AllowAnonymousMeetingParticipantsToAccessWhiteboards <SharingState>]
+ [-AllowCommentsTextOnEmailEnabled <Boolean>]
+ [-AllowDownloadingNonWebViewableFiles <Boolean>]
+ [-AllowEditing <Boolean>]
+ [-AllowEveryoneExceptExternalUsersClaimInPrivateSite <Boolean>]
+ [-AllowGuestUserShareToUsersNotInSiteCollection <Boolean>]
+ [-AllowOverrideForBlockUserInfoVisibility]
+ [-AllowSensitivityLabelOnRecords <Boolean>]
+ [-AllowSharingOutsideRestrictedAccessControlGroups <Boolean>]
+ [-AllowWebPropertyBagUpdateWhenDenyAddAndCustomizePagesIsEnabled <Boolean>]
+ [-AnyoneLinkTrackUsers <Boolean>]
+ [-AppBypassInformationBarriers <Boolean>]
+[-ApplyAppEnforcedRestrictionsToAdHocRecipients <Boolean>]
+ [-AutofillColumnScope <SyntexFeatureScopeValue>]
+ [-AutofillColumnsSelectedSitesList [String[]]]
+ [-AutofillColumnsSelectedSitesListOperation <SelectedSitesListOperations>]
  [-BccExternalSharingInvitations <Boolean>]
  [-BccExternalSharingInvitationsList <String>]
+ [-BlockDownloadFileTypeIds <SPBlockDownloadFileTypeId[]>]
```

Comment:
I think it should be [SPBlockDownloadFileTypeId[]] instead of <SPBlockDownloadFileTypeId[]>.
I am not sure whether this is what you had touched or not.